dragome-w3c-standards/src/main/java/org/w3c/dom/html/CanvasPixelArray.java

```diff
@@ -6,6 +6,6 @@
 {
     // CanvasPixelArray
     public int getLength();
-    public byte getElement(int index);
-    public void setElement(int index, byte value);
+    public int getElement(int index);
```

Comment:
This code is generated automatically from webidl specifications, cannot be modified this way. We could change it webidl if it's not matching the w3c standard and if not, we should create an extension interface of CanvasPixelArray to allow the addition of these new helper methods.